### PR TITLE
feat- announcement banner

### PR DESCRIPTION
**Pull request by IamDikshaR**

### CODE REVIEW

File: README.md
Comment (tanishdotdev) [inline README.md:1]: Add a new line at the bottom of all files.

File: src/atoms-diksha/buttons/bordered/index.astro
Comment (tanishdotdev) [inline src/atoms-diksha/buttons/bordered/index.astro:1]: Remove the paragraph and add those classes in the button only.

File: src/atoms-diksha/buttons/solid/index.astro
Comment (tanishdotdev) [inline src/atoms-diksha/buttons/solid/index.astro:1]: Remove the paragraph and add those classes in the button only.

File: src/atoms-diksha/paragraphs/p-nav/index.astro
Comment (tanishdotdev) [inline src/atoms-diksha/paragraphs/p-nav/index.astro:1]: 1. This should be formatted in 3 lines and add a new line at the end.
2. Rename this file, src/atoms-diksha/paragraphs/sm/index.astro

File: src/components/common/announcement-banner/index.astro
Comment (tanishdotdev) [inline src/components/common/announcement-banner/index.astro:1]: 1. Remove the text and font classes in anchor tag. Use span or paragraph instead.

File: src/components/common/header/index.astro
Comment (tanishdotdev) [inline src/components/common/header/index.astro:1]: 1. Add an empty line above and bottom of imports. Line2 and line15.
2. Keep this in a single line <header class="sticky top-0 z-10 flex justify-between items-center py-2 px-16 sm:px-8 bg-color-bg">
3. Create h3 atom.
4. Keep this in a single line <button
          class="material-symbols-outlined font-light text-color-primary cursor-pointer"
          >keyboard_arrow_down</button
        >
5. Remove text classes from <a
                href="$"
                class="font-poppins text-sm font-medium whitespace-nowrap"
              >
Use span or paragraph.

File: src/components/home/hero/index.astro
Comment (tanishdotdev) [inline src/components/home/hero/index.astro:1]: 1. Keep the section in one line.
2. Use picture internal atom and define size in div wrapping the picture atom.
3. Create h3 atom.
4. Define the color in tailiwind config file. text-[#febeac]
5. Create paragraph atom.
6. Keep paragraph, div, sections in one line only.

File: src/components/home/professional-marquee/index.astro
Comment (tanishdotdev) [inline src/components/home/professional-marquee/index.astro:1]: 1. Add an empty line above and below like:
---

const profession = [
  "Researchers",
  "Managers",
  "Content creators",
  "Entrepreneurs",
  "ADHD Coach",
]

---
2. Create h3 atom.
3. Keep the div in one line.
4. Use indent of 4 spaces in one tab.
5. Format the map function correctly.
6. Add an empty line at the bottom of the file in all files.

File: src/layouts/layout/index.astro
Comment (tanishdotdev) [inline src/layouts/layout/index.astro:1]: 1. Format the props in multiple lines.
2. Font import link should be kept in single line.

File: src/pages/index.astro
Comment (tanishdotdev) [inline src/pages/index.astro:1]: Why have you made changes in this file, Was it needed?

File: tailwind.config.mjs
Comment (tanishdotdev) [inline tailwind.config.mjs:1]: Use proper indentation in all files. 4 spaces.

**Review by tanishdotdev — CHANGES_REQUESTED** [review]

File: README.md
Comment (IamDikshaR) [reply README.md:1]: done

File: tailwind.config.mjs
Comment (IamDikshaR) [reply tailwind.config.mjs:1]: done

File: src/atoms-diksha/buttons/bordered/index.astro
Comment (IamDikshaR) [reply src/atoms-diksha/buttons/bordered/index.astro:1]: done

File: src/atoms-diksha/buttons/solid/index.astro
Comment (IamDikshaR) [reply src/atoms-diksha/buttons/solid/index.astro:1]: done

File: src/atoms-diksha/paragraphs/p-nav/index.astro
Comment (IamDikshaR) [reply src/atoms-diksha/paragraphs/p-nav/index.astro:1]: done

File: src/components/common/announcement-banner/index.astro
Comment (IamDikshaR) [reply src/components/common/announcement-banner/index.astro:1]: done

File: src/components/common/header/index.astro
Comment (IamDikshaR) [reply src/components/common/header/index.astro:1]: done

File: src/components/home/hero/index.astro
Comment (IamDikshaR) [reply src/components/home/hero/index.astro:1]: done

File: src/components/home/professional-marquee/index.astro
Comment (IamDikshaR) [reply src/components/home/professional-marquee/index.astro:1]: done

File: src/layouts/layout/index.astro
Comment (IamDikshaR) [reply src/layouts/layout/index.astro:1]: done

File: src/pages/index.astro
Comment (IamDikshaR) [reply src/pages/index.astro:1]: I actually opened the file to check for something and my formatter auto-formatted and made changes to the code here

**Pull request closed without merging**